pandas/tests/arrays/categorical/test_repr.py

```diff
@@ -147,8 +147,6 @@ def test_categorical_repr_datetime(self):
         idx = date_range("2011-01-01 09:00", freq="H", periods=5)
         c = Categorical(idx)
 
-        # TODO(wesm): exceeding 80 characters in the console is not good
-        # behavior
```

Comment:
is this referring to the code or to the string being returned?  black only affects the former

Reply:
It's about the output.

```python
In [16]: idx = pd.CategoricalIndex(list(string.ascii_letters))

In [17]: [len(x) for x in repr(idx).split('\n')]
Out[17]: [77, 77, 77, 77, 38, 107]
```